lib/gist.rb

```diff
@@ -127,7 +127,8 @@ def multi_gist(files, options={})
 
     existing_gist = options[:update].to_s.split("/").last
     if options[:anonymous]
-      access_token = nil
+      raise 'Anonymous gist support has been removed by Github. ' \
+        'See: https://blog.github.com/2018-02-18-deprecation-notice-removing-anonymous-gist-creation/'
```

Comment:
@estebanz01 Thanks for this — can we make this error explain to the user what to do about it?

```
raise 'Please log in with `gist --login`. (Github now requires credentials to gist https://bit.ly/2GBBxKw)'
```